cmd/argoexec/commands/agent.go

```diff
@@ -103,6 +103,10 @@ func initAgentExecutor() *executor.AgentExecutor {
 	if !ok {
 		log.Fatalf("Unable to determine workflow name from environment variable %s", common.EnvVarWorkflowName)
 	}
+	workflowUid, ok := os.LookupEnv(common.EnvVarPodUID)
```

Comment:
minor - but I don't think you need to error here, just ignore

Reply:
this looks wrong to me, why `EnvVarPodUID` not `EnvVarWorkflowUID`, maybe shows why test is important